proposals/first-class-span-types.md

```diff
@@ -50,6 +50,9 @@ An implicit span conversion permits `array_types`, `System.Span<T>`, `System.Rea
 
 ------
 
+Any Span/ReadOnlySpan types are considered applicable for the conversion if they match by their fully-qualified name.
+If they do not match the types that are selected by the compiler's well-known type resolution logic, a compile-time error is reported.
```

Comment:
This is not what we decided? We won't check that, and we can't really specify it either, since "the compiler's well-known type resolution logic" isn't a thing in the spec.

Reply:
Ah, okay, I thought that was part of the option we discussed, but I might have misunderstood.

Reply:
Then the question is how will we emit - do we need to find the operators on the Span types manually? It would be much easier to just use the well-known member machinery; and hence report an error if the Span type doesn't match the well-known type. Maybe that doesn't have to be specified, it can be an implementation detail like well-known types are.